Formula/c/criterion.rb

```diff
@@ -25,10 +24,12 @@ class Criterion < Formula
   depends_on "pkg-config" => :build
   depends_on "libgit2"
   depends_on "nanomsg"
+  depends_on "nanopb"
+
   uses_from_macos "libffi"
 
   def install
-    system "meson", "setup", *std_meson_args, "--force-fallback-for=boxfort", "build"
+    system "meson", "setup", *std_meson_args, "--force-fallback-for=boxfort,debugbreak,klib", "build"
```

Comment:
we actually have `debugbreak` formula

Reply:
It won't work since there is no `.pc`/`.cmake`/etc files - https://mesonbuild.com/Dependencies.html#dependency-detection-method